src/krux/camera.py

```diff
@@ -142,3 +163,41 @@ def capture_qr_code_loop(self, callback, x_offset=False):
         if parser.is_complete():
             return (parser.result(), parser.format)
         return (None, None)
+
+    def capture_entropy(self, callback):
```

Comment:
Yeah, this is too naive to use imo. Would be better to build entropy by having the user take a series of captures like they do a series of dice rolls. Perhaps even a MIN_CAPTURES analogous to MIN_ROLLS, if we assume each image is 10 bits of entropy...

Though ideally we'd be trying to count the entropy in each captured image.

Reply:
Makes no sense for me. Each pixel has 16 bits of information, its entropy may be low quality, non linear, with possible saturated regions, etc., but there are 76.800 of them, in a raw, uncompressed image.